modules/meta/arrow-meta/src/main/java/arrow/extensions/ExtensionProcessor.kt

```diff
@@ -233,7 +233,7 @@ class ExtensionProcessor : MetaProcessor<extension>(extension::class), PolyTempl
     companionOrFactory: TypeName,
     impl: String
   ): String =
-    """|return ${+companionOrFactory}.${typeClass.name.simpleName.decapitalize()}${+instance.typeVariables}(${requiredParameters.code { +it.name }}).run {
+    """|return ${+companionOrFactory}.${typeClass.name.simpleName.decapitalize()}${+instance.typeVariables}(${requiredParameters.code { +it.name }}).run·{
```

Comment:
Ese puntito que es lo que ess??

Reply:
·???

Reply:
It's the way to specify non-breaking spaces for latest KotlinPoet versions so it doesn't insert a new line on that space.

Reply:
Longer explanation: https://github.com/square/kotlinpoet#spaces-wrap-by-default